.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Java CI build and test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        java: [16-ea]
+      fail-fast: false
+      max-parallel: 4
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Verify with Maven
+        run: mvn -X verify -B --file pom.xml
```

Comment:
Not sure about `-X` - that's very verbose. And `-B --file pom.xml` is new to me. Why not just `mvn verify`? (Honest curiosity.)

Reply:
@nipafx yes, I forgot to drop the -X. When I am helping others and forking projects I tend to get surprised with odd build problems. Also, regarding just mvn verify is fine. Sometimes people like the formal maven switches to be more explicit (for the mvn snobs out there ;-).

Reply:
I fixed it (removed -X, etc.). It should be in the commit, but if not I can make another PR if you prefer.